### ISSUE TITLE
[Bug] Table selector fails on Heasarc

### ISSUE THREAD

**Issue by lmichel** [lmichel/TAP-complex-data#150]
**Describe the bug**
service : //heasarc.gsfc.nasa.gov/xamin/vo/tap

the table selector does not work likely due to some bad encoded char in the query (`;&#xA; ` likely)
<img width="1079" alt="Screenshot 2022-01-25 at 23 04 19" src="https://user-images.githubusercontent.com/11462648/151067345-8bc4a9c1-8b0a-4e9f-99d3-99cb34001305.png">

Th issue might go away if the \n is preceded by a space.

**Comment by galyfray** [comment]
almost everything doesn't work with this service it doesn't handle even simple joins ...

**Comment by lmichel** [comment]
When you run the table selector on that service with e.g.`EIN` as string pattern, you execute the following query string
```
SELECT TOP 500 TAP_SCHEMA.tables.table_name AS noField, TAP_SCHEMA.tables.schema_name AS schemaName, TAP_SCHEMA.tables.description AS tableDesc, TAP_SCHEMA.schemas.description AS schemDesc, TAP_SCHEMA.tables.table_name AS tableName FROM TAP_SCHEMA.tables WHERE ( ( UPPER(TAP_SCHEMA.tables.table_name) LIKE '%EIN%' ) )
```

The is a missing join between `TAP_SCHEMA.tables` and `TAP_SCHEMA.schemas`

**Comment by galyfray** [comment]
yes that's somehow half expected. As said earlier this services doesn't event support simple joins thus the API register no joins at all. The search bar can't work properly this way and thus should be disabled in such cases

**Comment by lmichel** [comment]
OK, but it runs wrong queries! 
in this case the search tool should be replaced with an alert like:  _Sorry GUY_

**Comment by lmichel** [comment]
still not working , error 400 in the console but nothing on the screen
![Screenshot 2022-02-03 at 13 37 38](https://user-images.githubusercontent.com/11462648/152344154-8589c652-7e62-4642-b17b-afa4449c14f3.png)
